src/utils/testing_utilities.cpp

```diff
@@ -64,4 +69,66 @@ namespace testingUtilities
         return ulpsDiff <= ulpsEpsilon;
     }
     // =========================================================================
-}
\ No newline at end of file
+
+  void wrapperEqual(int i, int j, int k, std::string dataSetName, 
```

Comment:
I've got a more generic version of this function already implemented that I will be submitting a PR for. See [`checkResults`](https://github.com/bcaddy/cholla/blob/790f7323e0882490f6f18227cef1e70a304916f7/src/utils/testing_utilities.cpp#L73)